src/modules/edc-demo/components/contract-definition-viewer/contract-definition-viewer.component.html

```diff
@@ -35,60 +33,78 @@
 
     <!-- Paginator -->
     <mat-paginator
+      *ngIf="contractDefinitionList.isReady"
       hidePageSize="true"
       pageSize="10"
-      [length]="contractDefinitions?.length"></mat-paginator>
+      [length]="
+        contractDefinitionList.data.numTotalContractDefinitions
+      "></mat-paginator>
   </div>
 
-  <div fxLayout="row wrap" fxLayoutAlign="start start">
-    <mat-card
-      *ngFor="let contractDefinition of contractDefinitions"
-      class="contract-definition-card">
-      <mat-card-header>
-        <mat-icon mat-card-avatar>policy</mat-icon>
-        <mat-card-title>{{ contractDefinition.id }}</mat-card-title>
-      </mat-card-header>
-      <mat-card-content>
-        <mat-list dense>
-          <mat-list-item>
-            <mat-icon mat-list-icon>policy</mat-icon>
-            <div class="policy-name" mat-line>Access policy</div>
-            <div mat-line>{{ contractDefinition.accessPolicyId }}</div>
-          </mat-list-item>
-          <mat-list-item>
-            <mat-icon mat-list-icon>policy</mat-icon>
-            <div class="policy-name" mat-line>Contract policy</div>
-            <div mat-line>{{ contractDefinition.contractPolicyId }}</div>
-          </mat-list-item>
-        </mat-list>
-
-        <mat-expansion-panel>
-          <mat-expansion-panel-header>
-            <mat-panel-title>Selectors</mat-panel-title>
-          </mat-expansion-panel-header>
+  <div class="flex">
```

Comment:
missing flex-direction class. always specify also "flex-row or flex-col".

Only exception is when the item has quite visibly only one child that will be centered or stretched making the direction of the flex irrelevant